cheatsheets/Error_Handling_Cheat_Sheet.md

```diff
@@ -1,16 +1,16 @@
 # Introduction
 
-Error handling is a part of the overall security of an application. Except in movies, an attack always begin by a **Reconnaissance** phase in which the attacker will try to gather as many technical information (often *name* and *version* properties) as possible about the target like for example the application server, the technologies, the frameworks, the libraries...
+Error handling is a part of the overall security of an application. Except in movies, an attack always begin by a **Reconnaissance** phase in which the attacker will try to gather as much technical information (often *name* and *version* properties) as possible about the target like, for example, the application server, the technologies, the frameworks, the libraries...
```

Comment:
The `...like, for example, the...` fragment doesn't read very well. It's not your change - it wasn't a great sentence before hand. Could you update this to something like `about the target, such as the application server, frameworks, libraries, etc.`